src/cucumber/java/stepdefinitions/TestStepDefinitions.java

```diff
@@ -13,6 +19,14 @@
  */
 public class TestStepDefinitions {
```

Comment:
This is very clean code however we should describe somewhere on this class or documentation that assertion run-time errors originate from utility classes. 

In general it is not a very good idea to seperate logic in this way as it makes it harder to read, even though common utility classes can make things easier... we lose readability of actually what is going on in the alter of simplicity.

Tradeoff in other words, we must document this somewhere, otherwise it looks like we are not using any assertions etc at first sight.